Formula/opencv.rb

```diff
@@ -1,14 +1,8 @@
 class Opencv < Formula
   desc "Open source computer vision library"
   homepage "https://opencv.org/"
-  url "https://github.com/opencv/opencv/archive/3.4.3.tar.gz"
-  sha256 "4eef85759d5450b183459ff216b4c0fa43e87a4f6aa92c8af649f89336f002ec"
-
-  bottle do
```

Comment:
These should stay here.

Reply:
I've updated opencv to `4.0.0` and renamed this one to `opencv@3.rb`.
Also I've created a symlink in to Aliases dir `opencv@4` to `opencv.rb` with opencv4 formula, and removed the old symlink `opencv@3`, but for some reason it's still picking it.